input/MergeStars/environment.yml

```diff
@@ -0,0 +1,127 @@
+name: enzo-e-analysis
```

Comment:
Should this file be part of the PR? Looks like a file that is produced during the build?